templates/web/pages/org_form/org_form.tt.html

```diff
@@ -10,6 +10,50 @@
 		<div class="panel" style="background-color:#ffdddd">The organization <strong>[% orgid %]</strong> does not exist yet. It will be created if you submit the form.</div>
 	[% END %]
 
+	[% IF org_members.size %]
```

Comment:
It does not work, I think you forgot to add org_members to the context for this template.

Reply:
I have added it in the #8511

Reply:
yes but, as you changed template, you have to move this line:
```perl
						$user_template_data_ref->{org_members} = \@org_members;
```